pkg/store/kotsstore/registry_store.go

```diff
@@ -38,18 +38,17 @@ func (s *KOTSStore) GetRegistryDetailsForApp(appID string) (registrytypes.Regist
 	}
 
 	registrySettings := registrytypes.RegistrySettings{
-		Hostname:    registryHostname.String,
-		Username:    registryUsername.String,
-		PasswordEnc: registryPasswordEnc.String,
-		Namespace:   registryNamespace.String,
-		IsReadOnly:  isReadOnly.Bool,
+		Hostname:   registryHostname.String,
+		Username:   registryUsername.String,
+		Namespace:  registryNamespace.String,
+		IsReadOnly: isReadOnly.Bool,
 	}
 
 	if !registryPasswordEnc.Valid {
 		return registrySettings, nil
 	}
 
-	decodedPassword, err := base64.StdEncoding.DecodeString(registrySettings.PasswordEnc)
```

Comment:
`registrySettings.PasswordEnc` is not used outside of this function, so it does not need to be part of this struct.